axiom/tests/test_txn.c

```diff
@@ -63,6 +63,21 @@ typedef struct {
   int line;
 } hash_is_subset_of_data_t;
 
+static bool need_to_stringify(const nrobj_t* val, const nrobj_t* obj) {
+  nrotype_t expected_type = nro_type(val);
+  nrotype_t found_type = nro_type(obj);
+
```

Comment:
How will this function behave if val or obj (or both) are NULL?

Reply:
I can add a check to return _false_ here if either object is NR_OBJECT_INVALID, or if either nrobj_t is NULL.
From there, the original program execution flow will take over as it falls into the _else_ clause and will handle those scenarios as it did before.